eng/scripts/Create-APIView.ps1

```diff
@@ -37,6 +37,27 @@ function Get-SwaggerReadMeFile {
     return $null
 }
 
+function Get-ImpactedTypeSpec {
```

Comment:
Could you make sure we cover all the previous cases? 
https://devdiv.visualstudio.com/DevDiv/_git/openapi-alps?path=/public/rest-api-specs-scripts/src/typespecAPIView.ts

Reply:
what do you mean previous cases? like how we determine which projects to include?
https://devdiv.visualstudio.com/DevDiv/_git/openapi-alps?path=/public/rest-api-specs-scripts/src/utils.ts&version=GBmain&line=1190&lineEnd=1206&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
The code and comments on alps manage if a PR is adding a new specification and therefore there are no baselines.

Reply:
Added in the scenario for new specs :)